test/error-cat.js

```diff
@@ -449,7 +449,14 @@ describe('ErrorCat', function() {
     it('should give empty data when none was provided', function(done) {
       error.report({});
       expect(rollbar.handleErrorWithPayloadData.firstCall.args[1])
-        .to.deep.equal({ custom: {} });
+        .to.deep.equal({ level: 'error', custom: {} });
```

Comment:
Perhaps you shouldn't implicitly test the default error level here and have an explicit test for it?